articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+```cs
+_spriteBatch.Draw(
+  _logo,                    // texture
+  new Vector2(              // position
+    (Window.ClientBounds.Width * 0.5f) - (_logo.Width * 0.5f),
+    (Window.ClientBounds.Height * 0.5f) - (_logo.Height * 0.5f)),
+  null,                     // sourceRectangle
+  Color.White,              // color
+  MathHelper.ToRadians(90), // rotation
+  Vector2.Zero,             // origin
+  1.0f,                     // scale
+  SpriteEffects.None,       // effects
+  0.0f);                    // layerDepth
+```
+
+The image should now be drawn centered on the game window and rotated so that it's vertical instead of horizontal right? Let's run it and find out.
+
+<figure><img src="./images/logo-rotated-offcenter.png" alt="Figure 4-10: Attempting to draw the MonoGame logo rotated 90° and centered on the game window."><figcaption><p><strong>Figure 4-10: Attempting to draw the MonoGame logo rotated 90° and centered on the game window.</strong></p></figcaption></figure>
+
+What happened here? We told it to draw it at the center position of the game window like before and to rotate it by 90°, which it did, but not exactly how we expected. So why did this happen? Recall from the parameters table above that the *origin* parameter specifies the point of origin in the texture where it is not only rendered from but also the point at which it's rotated from. In the code sample above, we used `Vector2.Zero` as the origin, which is the default origin point, or the top-left corner of the image. So when we told it to rotate 90°, it did so from the top-left corner of the image. Let's fix the code so that the origin is now in the center of the image.
+
+```cs
+_spriteBatch.Draw(
+  _logo,                    // texture
+  new Vector2(              // position
+    Window.ClientBounds.Width, 
+    Window.ClientBounds.Height) * 0.5f,
+  null,                     // sourceRectangle
+  Color.White,              // color
+  MathHelper.ToRadians(90), // rotation
+  new Vector2(              // origin
+    _logo.Width, 
+    _logo.Height) * 0.5f,
+  1.0f,                     // scale
+  SpriteEffects.None,       // effects
+  0.0f);                    // layerDepth
+```
+
+In the code above, we have adjust the *origin* parameter to use the center point of the texture. This will be the origin point at which it will now rotate from. It is also the origin point at which it is rendered from, so we no longer need to subtract half the logo width and height from the *position* parameter. After making this change, if we run the game now, we'll see it drawn correctly rotated 90° and centered on the game window.
+
+<figure><img src="./images/logo-rotated-centered.png" alt="Figure 4-11: The MonoGame logo drawn rotated 90° and centered on the game window."><figcaption><p><strong>Figure 4-11: The MonoGame logo drawn rotated 90° and centered on the game window.</strong></p></figcaption></figure>
+
+## Drawing Texture Regions

```

Comment:
Love this section, I don't think I have seen it gone into this depth previously

Reply:
Thanks :D